src/backend/distributed/commands/function.c

```diff
@@ -1061,6 +1072,63 @@ PlanDropFunctionStmt(DropStmt *stmt, const char *queryString)
 }
 
 
+List *
+PlanAlterFunctionDependsStmt(AlterObjectDependsStmt *stmt, const char *queryString)
```

Comment:
Missing function comment, few others as well.

We also don't use `queryString` so maybe not pass at all.

Reply:
will add the comments.

will respond on the `queryString` part in the subsequent comment as they are very much aligned.